functests/utils/mcps/mcps.go

```diff
@@ -89,6 +89,18 @@ func GetConditionStatus(mcpName string, conditionType machineconfigv1.MachineCon
 	return corev1.ConditionUnknown
 }
 
+// GetConditionReason return the message of the given MCP
```

Comment:
nit: of the MCP's condition of given type

Reply:
Removed this func.